chromaui/chromatic-e2e
# Bump SB Dependencies

Issue: #

## What Changed

We bumped some of the underlying dependencies.

## How to test


1. Check CI for no failures.
2. Run a build using canary for Cypress.
3. Run a build using canary for Playwright.
